test/utility.spec.js

```diff
@@ -0,0 +1,15 @@
+import { jest } from '@jest/globals';
+import * as utility from '../src/utility';
+
+describe('toggleShowElement', () => {
+  const mockElement = {
+    classList: {
+      toggle: jest.fn(),
+    },
+  };
+
+  it('should call toggle with "hide"', () => {
+    utility.toggleShowElement(mockElement);
+    expect(mockElement.classList.toggle).toBeCalledWith('hide');
+  });
+});
```

Comment:
You need to write more test in this file, you only wrote a test for just one method which is `toggleShowElement `, kindly write test for the remaining methods in ``utility.js.